lib/api/utilities/endpoints/delete.rb

```diff
@@ -67,6 +69,7 @@ def process(current_user, instance)
         def render(call)
           if success?(call)
             yield
+            present_success(call)
```

Comment:
We might want to extract this into something (e.g. module) shared between `Bodied` and `Delete` end points as `Bodied` has the exact same method implementation. But I can also do that in a later PR.

Reply:
It could also be a superclass of `Bodied`. We could then also move parts of the initializer and the `deduce...` method into it.